bundle/metadata/annotations.yaml

```diff
@@ -5,7 +5,6 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: numaresources-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.bundle.channel.default.v1: alpha
```

Comment:
@fromanirh I remember you added it, probably we should add some parameter to avoid override by `make bundle`

Reply:
good point, need to check why is being deleted

Reply:
handled here: https://github.com/openshift-kni/numaresources-operator/pull/75 
not blocking for this PR